src/controllers/__tests__/sessionController.test.ts

```diff
@@ -30,16 +30,18 @@ const mockSmembers = jest.fn();
 const mockSismember = jest.fn();
 const mockSrem = jest.fn();
 const mockTtl = jest.fn();
-const mockRedisClient = createMockRedis().createClient();
-mockRedisClient.get = mockGet;
-mockRedisClient.mget = mockMget;
-mockRedisClient.smembers = mockSmembers.mockImplementation((_, callback) => {
-  callback(null, [`SESSIONINFO-${mockedUser.session_token}`]);
-});
-mockRedisClient.sismember = mockSismember;
-mockRedisClient.ttl = mockTtl;
-mockRedisClient.set = mockSet;
-mockRedisClient.srem = mockSrem;
+const mockRedisClient = ({
+  get: mockGet,
+  mGet: mockMget,
+  sMembers: mockSmembers.mockImplementation(_ =>
+    Promise.resolve([`SESSIONINFO-${mockedUser.session_token}`])
+  ),
+  sIsMember: mockSismember,
+  ttl: mockTtl,
+  set: mockSet,
+  sRem: mockSrem,
+  setEx: mockSet
```

Comment:
Use a specific mock function for `setEx` method.

Reply:
solved in https://github.com/pagopa/io-backend/pull/962/commits/5b8820ceb39b4348a366669e07beb1dda8c2db26